LutronLuxmeterIntegration/react/src/App.tsx

```diff
@@ -0,0 +1,139 @@
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import styled from '@mui/material/styles/styled';
+import CssBaseline from '@mui/material/CssBaseline';
+
+import { Nav } from './components/Nav';
+import { ContainerLoader } from './components/ContainerLoader';
+import { TServerData, TServerConvertedData, serverDataSchema } from './models/schema';
+import { useInitializeOnMount } from './hooks/useInitializeOnMount';
+import { ZodError } from 'zod';
+import { useSnackbar } from './hooks/Snackbar';
+import { Typography } from '@mui/material';
+import { Headline } from './components/Title';
+import { FormWrapper } from './FormWrapper';
+import { mockedSettings } from './models/mock';
+
+export const App = () => {
+    const { displaySnackbar } = useSnackbar();
+
+    const [defaultValues, setDefaultValues] = useState<TServerData | null>(null);
+
+    const convert = (fetched: TServerConvertedData) => {
```

Comment:
Ty konverze tam a zpatky mi prijdou zbytecne komplikovane, kdyz stacilo ty dve podminky dat primo do skriptu, ale asi je to jedno.

Reply:
Příště se poučím.